### PR TITLE
Issue 1 view page

### PR DESCRIPTION
**Pull request by sarah-you**
screen recording confirming the webpage functionality -- sent via slack due to file size exceeding limit

### CODE REVIEW

File: js/main.js [hunk 1]
```diff
@@ -0,0 +1,206 @@
+// api
+function getAnimeData(name) {
+  const xhr = new XMLHttpRequest();
+  const $list = document.querySelector('#anime-list');
+
+  xhr.open('GET', 'https://api.jikan.moe/v4/top/anime');
+  xhr.responseType = 'json';
+  xhr.addEventListener('load', function () {
+    // console.log('status: ', xhr.status);
+    // console.log('response: ', xhr.response);
```
Comment (shawnkost) [inline js/main.js]: Remove all old code

File: js/main.js [hunk 2]
```diff
@@ -0,0 +1,206 @@
...
+      $modalAddBtn.textContent = '+ Add';
+      $modalAddBtn.setAttribute('class', 'add modal-btn');
+      $modalBtnWrap.appendChild($modalAddBtn);
+
+      const $modalCloseBtn = document.createElement('button');
+      $modalCloseBtn.textContent = 'Close';
+      $modalCloseBtn.setAttribute('class', 'close modal-btn');
+      $modalBtnWrap.appendChild($modalCloseBtn);
+      // end of modal elements
+    }
```
Comment (shawnkost) [inline js/main.js]: This should all be moved into it's own function, rather than all in this `load` event listener.

File: js/main.js [hunk 3]
```diff
@@ -0,0 +1,206 @@
...
+
+    // modal pop up if user clicks on a show (added a conditional bc clicking outside the list showed error, since the event listener was on the ol and buttons in the modal are still part of the ol)
+    $ol.addEventListener('click', function openModal(event) {
+      if (event.target.tagName !== 'OL' && event.target.tagName !== 'LI' && event.target.tagName !== 'BUTTON') {
+        const $parentElement = event.target.closest('.list-wrap');
+        const $closestModal = $parentElement.nextElementSibling;
+        $defaultModal.className = 'main-container modal-opened';
+        $closestModal.className = 'show modal';
+      }
+    });
```
Comment (shawnkost) [inline js/main.js]: This should be outside of any functions

File: js/main.js [hunk 4]
```diff
@@ -0,0 +1,206 @@
...
+
+      const $modalCloseBtn = document.createElement('button');
+      $modalCloseBtn.textContent = 'Close';
+      $modalCloseBtn.setAttribute('class', 'close modal-btn');
+      $modalBtnWrap.appendChild($modalCloseBtn);
+      // end of modal elements
+    }
+
+    // modal event listeners
+    const $defaultModal = document.querySelector('.main-container');
```
Comment (shawnkost) [inline js/main.js]: This should go at the top of the file

File: js/main.js [hunk 5]
```diff
@@ -0,0 +1,206 @@
...
+    const $modal = document.querySelectorAll('.modal');
+    const $ol = document.querySelector('#anime-list');
```
Comment (shawnkost) [inline js/main.js]: Line 181 should go at the top of the file

File: js/main.js [hunk 6]
```diff
@@ -0,0 +1,206 @@
...
+        $defaultModal.className = 'main-container';
+        $modal[i].className = 'hidden modal';
+      });
+    // end of modal event listeners
+    }
+  });
+  xhr.send();
+}
+
+getAnimeData();
```
Comment (shawnkost) [inline js/main.js]: I would call this inside of an event listener instead. Like `DOMContentLoaded`

**Review by shawnkost — CHANGES_REQUESTED** [review]
Looking good so far Sarah! A few changes requested here

File: js/main.js (hunk 6 above)
Comment (sarah-you) [reply js/main.js]: put `xhr.send()` and `getAnimeData` function  inside `window event listener` with type `DOMContentLoaded`

File: js/main.js (hunk 2 above)
Comment (sarah-you) [reply js/main.js]: moved outside to its own function `hiddenModalDOM()`; 
called inside the `load `event listener

File: js/main.js (hunk 3 above)
Comment (sarah-you) [reply js/main.js]: moved this outside of any functions

**Review by shawnkost — APPROVED** [review]
👍🏻

**Pull request merged**